packages/web-client/app/routes/boom.ts

```diff
@@ -0,0 +1,11 @@
+import Controller from '@ember/controller';
+import Route from '@ember/routing/route';
+
+export default class BoomRoute extends Route {
+  setupController(controller: Controller, model: any, transition: any) {
+    super.setupController(controller, model, transition);
+
+    //@ts-ignore intentional error
+    raiseAnUnhandledExceptionOnPurpose();
+  }
+}
```

Comment:
Is this meant to be a permanent inclusion? I thought maybe it would be just for testing to ensure Sentry received something.

Reply:
Its purpose is to verify Sentry functionality. I want to verify it on staging and prod as well. I'm inclined to leave it there after that to make verifying any sentry config changes easy but am open to other views.